src/test/java/seedu/address/testutil/TypicalStudent.java

```diff
@@ -46,17 +48,20 @@
             .withEmail("anna@example.com").build();
 
     // Manually added
-    public static final Student HOON = new StudentBuilder().withName("Hoon Meier").withTelegram("8482424")
+    public static final Student HOON = new StudentBuilder().withUuid("bd4f4c43-35d5-4afc-ab83-88cf20314560")
+            .withName("Hoon Meier").withTelegram("8482424")
             .withEmail("stefan@example.com").build();
-    public static final Student IDA = new StudentBuilder().withName("Ida Mueller").withTelegram("8482131")
+    public static final Student IDA = new StudentBuilder().withUuid("7f551343-cadb-4df3-8b50-76394fe84107")
+            .withName("Ida Mueller").withTelegram("8482131")
             .withEmail("hans@example.com").build();
 
     // Manually added - Student's details found in {@code CommandTestUtil}
-    public static final Student AMY = new StudentBuilder().withName(VALID_NAME_AMY).withTelegram(VALID_TELEGRAM_AMY)
+    public static final Student AMY = new StudentBuilder().withUuid(VALID_UUID_AMY)
+            .withName(VALID_NAME_AMY).withTelegram(VALID_TELEGRAM_AMY)
             .withEmail(VALID_EMAIL_AMY).withTags(VALID_TAG_EXPERIENCED).build();
-    public static final Student BOB = new StudentBuilder().withName(VALID_NAME_BOB).withTelegram(VALID_TELEGRAM_BOB)
-            .withEmail(VALID_EMAIL_BOB)
-            .withTags(VALID_TAG_AVERAGE, VALID_TAG_EXPERIENCED).build();
+    public static final Student BOB = new StudentBuilder().withUuid(VALID_UUID_BOB)
+            .withName(VALID_NAME_BOB).withTelegram(VALID_TELEGRAM_BOB)
+            .withEmail(VALID_EMAIL_BOB).withTags(VALID_TAG_AVERAGE, VALID_TAG_EXPERIENCED).build();
 
     public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
```

Comment:
perhaps this should start with a lowercase?
```suggestion
    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // a keyword that matches MEIER
```